src/libs/reportUtils.js

```diff
@@ -220,6 +220,21 @@ function isDeletedAction(action) {
     return action.message.length === 0 || action.message[0].html === '';
 }
 
+/**
+ * Used before merging Report data.
+ * For now cut last message text to specific length, because we don't need full last message
+ *
+ * @param {String} lastMessageText
+ * @returns {String}
+ */
+function formatReportLastMessageText(lastMessageText) {
+    if (_.isString(lastMessageText) && lastMessageText.length > CONST.REPORT.LAST_MESSAGE_TEXT_MAX_LENGTH) {
+        return lastMessageText.substr(0, CONST.REPORT.LAST_MESSAGE_TEXT_MAX_LENGTH);
+    }
+
+    return lastMessageText;
```

Comment:
```suggestion
    return String(lastMessageText).substring(0, CONST.REPORT.LAST_MESSAGE_TEXT_MAX_LENGTH);
```
![image](https://user-images.githubusercontent.com/24370807/149091528-ad8eb816-1be9-46f1-887d-68a486742de5.png)